src/main.rs

```diff
@@ -32,23 +38,39 @@ fn main() {
 fn run(path: &String) -> Result<(), parser::ParserError> {
```

Comment:
I think we should use some library to parse arguments, in the future It'll be easy to add some new logic to cli, I'd like to add passing Globs instead of only one file, it'll be useful for big projects

Reply:
yes this should be the objective of future milestones, for now the goal is to successfuly transpile single file ( minor errors for more comlicated stuff is expected)